charts/trino/templates/configmap-catalog.yaml

```diff
@@ -1,16 +0,0 @@
-{{- if or .Values.catalogs .Values.additionalCatalogs }}
```

Comment:
This is a breaking change, and it doesn't look required - you could use either a configmap or secrets.